app/cmd/root.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright © 2022 Merbridge Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"runtime"
+	"strings"
+	"syscall"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/merbridge/merbridge/app/cmd/options"
+	"github.com/merbridge/merbridge/config"
+	"github.com/merbridge/merbridge/internal/ebpfs"
+	"github.com/merbridge/merbridge/internal/pods"
+)
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "mbctl",
+	Short: "Use eBPF to speed up your Service Mesh like crossing an Einstein-Rosen Bridge.",
+	Long:  `Use eBPF to speed up your Service Mesh like crossing an Einstein-Rosen Bridge.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		client := options.NewOptions()
+		w := pods.NewWatcher(client)
+
+		_ = w.Start()
+		log.Info("Pod Watcher Ready")
```

Comment:
this line conflicts with https://github.com/merbridge/merbridge/blob/3f10d6f82efa290d6b31ddfcfff6b6e0762c7c7c/.github/workflows/e2e.yaml#L53

Reply:
which cause e2e test timeout.